app/controllers/comments_controller.rb

```diff
@@ -8,14 +8,20 @@ class CommentsController < ApplicationController
   PATH_PARENTS = []
 
   def create
-    comment = Comment.new(comment_params)
+    @comment = Comment.new(comment_params)
 
-    if (comment.text.present? && comment.save)
-      anchor = ActionView::RecordIdentifier.dom_id(comment)
+    if (@comment.text.present? && @comment.save)
+      anchor = ActionView::RecordIdentifier.dom_id(@comment)
     else
       flash[:alert] = "Oh no! We can't save your comment. Please try again?"
     end
-    redirect_to commentable_path(comment.commentable, anchor || nil)
+
+    @new_comment = self.class::PATH_PARENTS + [Comment.new(commentable: @comment.commentable)]
+
+    respond_to do |format|
+      format.html { redirect_to commentable_path(@comment.commentable, anchor || nil) }
```

Comment:
Yes, the CommentsController is used all over the app, fore example projects are comment-able:
https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/app/views/projects/show.html.slim#L81

The following models seem support comments:
- Application
- Activity
- Project
- Team